ketch-ios/iOS Ketch Pref Center using SwiftUI/README.md

```diff
@@ -1,370 +1,155 @@
-struct WebView: UIViewRepresentable {
-    @Binding var isPresented: Bool
-    @Binding var identities: [Identity]
-    let htmlFileName: String
+...
 
-    ...
+private var config: ConsentConfig?
 
-}
-```
-
-- Then at the bottom of the file, create an extension method to add a query parameter to an existing url.
-```swift
-extension URL {
-
-    func appending(_ queryItem: String, value: String?) -> URL {
-        guard var urlComponents = URLComponents(string: absoluteString) else { return absoluteURL }
-        var queryItems: [URLQueryItem] = urlComponents.queryItems ??  []
-        let queryItem = URLQueryItem(name: queryItem, value: value)
-        queryItems.append(queryItem)
-        urlComponents.queryItems = queryItems
-        return urlComponents.url!
-    }
-}
-```
-
-- Update the `updateUIView` function on the `WebView` struct to iterate over the array of identities and add them to the HTML page's URL.
-```swift
-struct WebView: UIViewRepresentable {
-
-    ...
-
-    func updateUIView(_ uiView: UIViewType, context: Context) {
-        let wv = uiView as? WKWebView
-        var url = Bundle.main.url(forResource: htmlFileName, withExtension: "html")!
-        for identity in identities {
-            url = url.appending(identity.code, value: identity.value)
-        }
-        wv?.load(URLRequest(url: url))
-    }
-
-    ...
-}
-```
-
-- Then add a bindable array to the `PreferenceView` struct to receive and pass along the identities to the `WebView`.
-```swift
-struct PreferenceView: View {
-    @Binding var isPresented: Bool
-    @Binding var identities: [Identity]
-    var body: some View {
-        WebView(isPresented: $isPresented, identities: $identities, htmlFileName: "index")
-    }
-}
-```
-
-- Return to the view using `PreferenceView`, create a state array containing the identities to use with the Ketch Preference Center, and pass the identities array into `PreferenceView`.
-```swift
 struct ContentView: View {
-    @State private var showingPopover = false
-    @State private var ids = [Identity(code: "visitorId", value: "user@test.com")]
+    @State private var configItem: ConsentConfig?
+
     var body: some View {
-        Text("Hello world!")
-            .padding()
-        Button("Show Preference Center") {
-            showingPopover = true
+        VStack {
+            Button("Show Preference Center") {
+                configItem = config
+            }
         }
-        .popover(isPresented: $showingPopover) {
-            PreferenceView(isPresented: $showingPopover, identities: $ids)
+        .onAppear {
+            ATTrackingManager.requestTrackingAuthorization { authorizationStatus in
+                if case .authorized = authorizationStatus {
+                    let advertisingId = ASIdentifierManager.shared().advertisingIdentifier
+
+                    config = ConsentConfig.configure(
+                        orgCode: "transcenda",
+                        propertyName: "website_smart_tag",
```

Comment:
Please update `orgCode` and `propertyName` values to use placeholders, like in the Storyboard README.